### PR TITLE
Adds `csdlVersion` param

### PR DESCRIPTION
**Pull request by irvinesunday**
Fixes https://github.com/microsoftgraph/msgraph-metadata/issues/282

This PR:
- Adds a new param: `csdlVersion` to be used in performing version-specific transforms. Default is set to v1.0: `v1_0`

### CODE REVIEW

File: transforms/csdl/transform.ps1 [hunk 1]
```diff
@@ -40,6 +42,7 @@ $outputFullPath = Get-PathWithPrefix -requestedPath $outputPath
 $xsltargs = [System.Xml.Xsl.XsltArgumentList]::new()
 $xsltargs.AddParam("remove-capability-annotations", "", $removeCapabilityAnnotations.ToString())
 $xsltargs.AddParam("add-innererror-description", "", $addInnerErrorDescription.ToString())
+$xsltargs.AddParam("csdlVersion", "", csdlVersion)
```
Comment (andrueastman) [inline transforms/csdl/transform.ps1:45]:
```suggestion
$xsltargs.AddParam("csdlVersion", "", $csdlVersion)
```

Comment (irvinesunday) [reply transforms/csdl/transform.ps1]: Resolved

File: transforms/csdl/transform.ps1 [hunk 2]
```diff
@@ -14,6 +14,8 @@ param (
     $removeCapabilityAnnotations = $true,
     [bool]
     $addInnerErrorDescription = $false
```
Comment (andrueastman) [inline transforms/csdl/transform.ps1:16]: I think we are also missing a trailing comma after this parameter.

**Review by baywet — CHANGES_REQUESTED** [review]
this probably needs to be updated as well as the open api validation script and the workflows.
https://github.com/microsoftgraph/msgraph-metadata/blob/master/scripts/run-metadata-validation.ps1

File: transforms/csdl/transform.ps1 (hunk 2 above)
Comment (irvinesunday) [reply transforms/csdl/transform.ps1]: Resolved.

**Comment by irvinesunday** [comment]
> this probably needs to be updated as well as the open api validation script and the workflows. https://github.com/microsoftgraph/msgraph-metadata/blob/master/scripts/run-metadata-validation.ps1

Effected with this PR: https://github.com/microsoftgraph/msgraph-metadata/pull/290

**Review by baywet — APPROVED** [review]

**Review by andrueastman — APPROVED** [review]

**Pull request merged**